src/overlays/actors/ovl_En_Owl/z_en_owl.c

```diff
@@ -128,15 +128,15 @@ void EnOwl_Init(Actor* thisx, PlayState* play) {
     }
 
     ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 36.0f);
-    SkelAnime_InitFlex(play, &this->skelAnime1, &object_owl_Skel_00C5F8, &object_owl_Anim_001ADC, this->jointTable1,
-                       this->morphTable1, 21);
-    SkelAnime_InitFlex(play, &this->skelAnime2, &object_owl_Skel_0105C0, &object_owl_Anim_00CDB0, this->jointTable2,
-                       this->morphTable2, 16);
+    SkelAnime_InitFlex(play, &this->skelAnime1, &gOwlFlyingSkel, &gOwlFlyAnim, this->jointTable1, this->morphTable1,
```

Comment:
Can use the limb max enums here. Also wouldn't be opposed to naming the joint and morph tables now (flying and perching instead of 1 and 2).